tool/gravity/cli/reconfigure.go

```diff
@@ -0,0 +1,152 @@
+/*
+Copyright 2020 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cli
+
+import (
+	"context"
+
+	"github.com/gravitational/gravity/lib/install"
+	installerclient "github.com/gravitational/gravity/lib/install/client"
+	"github.com/gravitational/gravity/lib/install/reconfigure"
+	"github.com/gravitational/gravity/lib/localenv"
+	"github.com/gravitational/gravity/lib/system/signals"
+	"github.com/gravitational/gravity/lib/utils"
+	"github.com/gravitational/gravity/lib/utils/cli"
+
+	"github.com/fatih/color"
+	"github.com/gravitational/trace"
+)
+
+const reconfigureMessage = `WARNING: This action will launch the operation of reconfiguring the cluster installed on this node to use a new advertise IP %v.
+Would you like to proceed? You can launch the command with --confirm flag to suppress this prompt in future.`
+
+// reconfigureCluster starts the cluster reconfiguration operation.
+//
+// Currently, the reconfiguration operation only allows to change advertise
+// address for single-node clusters.
+func reconfigureCluster(env *localenv.LocalEnvironment, config InstallConfig, confirmed bool) error {
+	// See if there's a cluster at all and whether it's running. The operation
+	// can be only performed when everything's stopped.
+	err := localenv.DetectCluster(env)
+	if err != nil && trace.IsNotFound(err) {
+		return trace.NotFound("The cluster doesn't appear to be installed on this node.")
+	}
+	if err == nil {
+		return trace.NotFound(`The cluster appears to be running on this node. Please stop all services using "gravity system stop" prior to initiating the operation.`)
```

Comment:
Why `NotFound` error here? I think `AlreadyExists` fits better (for the lack of `AlreadyRunning`).